include/common_structs.h

```diff
@@ -123,7 +123,7 @@ typedef struct Npc {
     /* 0x078 */ s32 shadowIndex;
     /* 0x07C */ f32 shadowScale;
     /* 0x080 */ s32 unk_80;
-    /* 0x084 */ char unk_84[4];
+    /* 0x084 */ s32 unk_84;
```

Comment:
undo this change

Reply:
This is probably why Jenkins doesn't like your PR too

Reply:
Wait you want me to have the char and [4] again?

Reply:
Yes